cmd/ocm/create/cluster/cmd.go

```diff
@@ -39,6 +41,17 @@ var args struct {
 	expirationTime    string
 	expirationSeconds time.Duration
```

Comment:
Can you make the expiration ones hidden, as we don't support these flags in production?